src/index.js

```diff
@@ -77,64 +78,72 @@ const handleInteraction = async ({ request, wait, sentry }) => {
 };
 
 // Process all requests to the worker
-const handleRequest = async ({ request, wait, sentry }) => {
+const handleRequest = ({ request, wait, sentry }) => {
     const url = new URL(request.url);
 
     // Send interactions off to their own handler
-    if (request.method === 'POST' && url.pathname === '/interactions')
-        return await handleInteraction({ request, wait, sentry });
+    if (request.method === 'POST' && url.pathname === '/interactions') return handleInteraction({
+        request, wait, sentry,
+    });
+
 
     // Otherwise, we only care for GET requests
-    if (request.method !== 'GET')
-        return new Response(null, { status: 404 });
+    if (request.method !== 'GET') return new Response(null, { status: 404 });
+
 
     // Health check route
-    if (url.pathname === '/health')
-        return new Response('OK', {
-            headers: {
-                'Content-Type': 'text/plain',
-                'Cache-Control': 'no-store, no-cache, must-revalidate, proxy-revalidate',
-                'Expires': '0',
-                'Surrogate-Control': 'no-store',
-            },
-        });
+    if (url.pathname === '/health') return new Response('OK', {
+        headers: {
+            'Content-Type': 'text/plain',
+            'Cache-Control': 'no-store, no-cache, must-revalidate, proxy-revalidate',
+            'Expires': '0',
+            'Surrogate-Control': 'no-store',
+        },
+    });
+
 
     // Privacy notice route
-    if (url.pathname === '/privacy')
-        return new Response(Privacy, {
-            headers: {
-                'Content-Type': 'text/plain',
-            },
-        });
+    if (url.pathname === '/privacy') return new Response(Privacy, {
+        headers: {
+            'Content-Type': 'text/plain',
+        },
+    });
+
 
     // Invite redirect
-    if (url.pathname === '/invite')
-        return redirectResponse(`https://discord.com/oauth2/authorize?client_id=${process.env.CLIENT_ID}&scope=applications.commands`);
+    // eslint-disable-next-line max-len
+    if (url.pathname === '/invite') return redirectResponse(`https://discord.com/oauth2/authorize?client_id=${process.env.CLIENT_ID}&scope=applications.commands`);
```

Comment:
Instead of using `// eslint-disable-next-line max-len`, the `ignoreStrings` opt should be used in the eslint config file if there are strings that go beyond the max length set.